.github/workflows/services-persist-tag.yml

```diff
@@ -1,30 +0,0 @@
-name: Persist service tag
```

Comment:
This is not used by any workflow anymore: both the `build-services` workflow and the new `frontend-workflow` directly commit the services versions.